components/kyma-operator/cmd/operator/main.go

```diff
@@ -128,3 +144,34 @@ func parseBackoffIntervals(backoffIntervals string) ([]uint, error) {
 
 	return backoffIntervalsParsed, nil
 }
+
+func setupLogrus(logFile string, overrideLogFormatToJSON bool) (*logrus.Logger, func(ch <-chan struct{}), error) {
+	// create the logger
+	logger := logrus.New()
+
+	if overrideLogFormatToJSON { // with Json Formatter
+		logger.SetFormatter(&logrus.JSONFormatter{})
+	}
+
+	//open the log file
+	file, err := os.OpenFile(logFile, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0755)
+	if err != nil {
+		return nil, nil, err
+	}
+	logger.SetOutput(file)
+	if logFile == STDOUT { //if the output file is /dev/stdout, we should not return a function to close the file
```

Comment:
I think we should just return a "NOOP" function.

Reply:
done